api/go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/bboughton/gcp-helpers v0.1.0
 	github.com/buger/jsonparser v1.1.1
 	github.com/caraml-dev/merlin-pyspark-app v0.0.3
+	github.com/caraml-dev/mlp v1.7.7-0.20230419034750-0b8e501b5331
```

Comment:
This needs to be updated to the new released version of mlp once https://github.com/caraml-dev/mlp/pull/84 is merged

Reply:
Updated to use commit from mlp main v1.7.7-0.20230420024248-2eaf397867ff